### ISSUE TITLE
dead link in docs

### ISSUE THREAD

**Issue by leuchtum** [powsybl/powsybl.github.io#326]
### Describe the current behavior

404 Error when going to

https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html

### Describe the expected behavior

---

### Describe the steps

Open https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html

### Environment

---

### Relevant Log Output

---

### Extra Information

---

**Comment by colinepiloquet** [comment]
Hi @leuchtum, actually the technical documentation was moved to https://powsybl.readthedocs.io/en/latest/.

This page is available here: https://powsybl.readthedocs.io/projects/powsybl-open-loadflow/en/latest/index.html.

**Comment by leuchtum** [comment]
@colinepiloquet explains a lot, thank you.

But in this case, the problem is even bigger. If you search for `https://www.powsybl.org/pages/documentation/` over the hole pypowsl organisatzion, you get 28 hits. Just have a look at the search:

https://github.com/search?q=org%3Apowsybl+https%3A%2F%2Fwww.powsybl.org%2Fpages%2Fdocumentation%2F&type=code

So that's 28 references to the old documentation, resulting in a 404 page.

**Comment by leuchtum** [comment]
Some of the 28 are valid, but others clearly not:

For example **valid**:
`https://www.powsybl.org/pages/documentation/developer/tutorials/` at https://github.com/powsybl/powsybl-diagram/blob/756d04da53ae865c482acdee12b1cbe078108418/README.md?plain=1#L61C49-L61C113

For example **invalid**:
`https://www.powsybl.org/pages/documentation/simulation/powerflow/`at https://github.com/powsybl/powsybl-entsoe/blob/6faac765480b45cf6480af30b7d5b79372a5d941/docs/flow_decomposition/configuration.md?plain=1#L20

**Comment by colinepiloquet** [comment]
Yes, you are right, thanks for pointing it out, I am trying to correct them all.

**Comment by colinepiloquet** [comment]
The tutorials are about to be moved too.

**Comment by leuchtum** [comment]
@colinepiloquet when you're at it, checkout this https://github.com/powsybl/.github/issues/57 issue I've opened right now too.
